### ISSUE TITLE
[BUG] last-child:mb-0 doesn't raise a warning about being invalid

### ISSUE THREAD

**Issue by sbdchd** [francoismassart/eslint-plugin-tailwindcss#305]
**Describe the bug**

```typescript
<div className="last-child:mb-0 mb-2" />
```

Doesn't raise a warning about being invalid, the correct name is `last:mb-0`

**To Reproduce**

Use the following css and find no error is raised
```typescript
<div className="last-child:mb-0 mb-2" />
```

**Expected behavior**
Raises error

**Screenshots**

**Environment (please complete the following information):**
 - OS: [e.g. macOS, windows 10]
 - Softwares + version used:
     -  [e.g. VSCode 1.54.3]
     -  [... Terminal 2.9.5, npm 6.14.5, node v14.5.0]

**Additional context**

**eslint config file or live demo**
By providing a link to a live demo, a demo video or a github repo fixing the issue will be easier.

**Comment by kachkaev** [comment]
This feature request is reflected in [README.md → Upcoming rules](https://github.com/francoismassart/eslint-plugin-tailwindcss#upcoming-rules) as `validate-modifiers`.

**Comment by francoismassart** [comment]
Fixed in v4, it will be released after cleaning up the existing issues.
